### DISCUSSION TITLE
Haseman R4D4 - Z-Wave, DIN Rail, 4 RELAY and 4 UNIVERSAL DIMMERS

### DISCUSSION THREAD

**Discussion by kimbo1st** [zwave-js/zwave-js-ui#4026] [General]
Wondering if this is templated and will work.
What I can read is that there is Fibaro inside… so should be?
http://z-wavelab.com/index.php?route=product/product&product_id=372

**Comment by robertsLando** [comment]
Do you have one included? If so you can check this by following the link you find next to the node id when you expand a row in control panel

cc @AlCalzone

**Comment by AlCalzone** [comment]
https://devices.zwave-js.io/?jumpTo=0x0115:0xf222:0x2111:0.0
